debian/kafka-connect-datagen/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Builds a docker image for Kafka Connect
+
+FROM confluentinc/cp-kafka-connect-base
+
+MAINTAINER devx@confluent.io
```

Comment:
I think we probably just want to be consistent with maintainership across the board (unless we really do dockerfile-specific team targeting), though I'm not sure why everything else is pointed at `partner-support@confluent.io`...

Reply:
Reverted to partner-support